datadog-logs-oci-orm/logs-function/func.py

```diff
@@ -0,0 +1,112 @@
+import io
+import os
+import json
+import logging
+import gzip
+import requests
+
+logger = logging.getLogger(__name__)
+
+DD_SOURCE = "oracle_cloud"  # Adding a source name.
+DD_SERVICE = "OCI Logs"  # Adding a service name.
+DD_TIMEOUT = 10 * 60  # Adding a timeout for the Datadog API call.
+DD_BATCH_SIZE = 1000  # Adding a batch size for the Datadog API call.
+
+
+def _should_compress_payload() -> bool:
+    return os.environ.get("DD_COMPRESS", "true").lower() == "true"
+
+
+def _compress_payload(payload: list[dict]):
```

Comment:
might be good to have ther return type defined here so it's clear this can return two different types of data